Backend/model/model.py

```diff
@@ -59,20 +61,71 @@ def location_validation(cls, location):
 
     @validator("pincode")
     def pincode_validation(cls, pincode):
-        if not len(pincode) == 6:
-            raise ValueError(
-                "This field should not be empty, less or greater than 6 digit")
+        for i in pincode:
+            if i.isdigit():
+                continue
+            else:
+                raise ValueError("Only digit accept")
         return pincode
 
 
 class Aanganwadi(BaseModel):
-    aanganwadiName: str
-    contactPersonName: str
-    contactPersonEmail: EmailStr
-    contactPersonPhone: int
-    contactPersonPassword: str
-    taluka: str
-    pincode: int
+    aanganwadiName: str = Field(..., min_length=2)
+    contactPersonName: str = Field(...)
+    contactPersonEmail: EmailStr = Field(...)
+    contactPersonPhone: str = Field(..., min_length=10, max_length=10)
+    contactPersonPassword: str = Field(...)
+    taluka: str = Field(...)
+    pincode: str = Field(..., min_length=6, max_length=6)
+
+    @validator("aanganwadiName")
+    def val(cls, aanganwadiName):
+        for i in aanganwadiName:
+            if i.isalpha():
+                continue
+            elif i == ' ':
+                continue
+            else:
+                raise ValueError("not accepted")
+        return aanganwadiName
+
+    @validator("contactPersonName")
+    def name_validation(cls, contactPersonName):
+        for i in contactPersonName:
+            if i.isalpha():
+                continue
+            elif i == ' ':
+                continue
+            else:
+                raise ValueError("only alphabet acceptable")
+        return contactPersonName.title()
+
+    @validator("contactPersonPhone")
+    def phone_number_validation(cls, contactPersonPhone):
+        for i in contactPersonPhone:
+            if i.isdigit():
+                continue
+            else:
+                raise ValueError("Only digit accept")
+        return contactPersonPhone
+
+    @validator("taluka")
+    def taluka_validation(cls, taluka):
+        for i in taluka:
+            if i.isalpha():
+                continue
+            else:
+                raise ValueError("Check entered value properly")
+        return taluka
+
+    @validator("pincode")
+    def pincode_validation(cls, pincode):
```

Comment:
Same here for all numeric only validations we can have only one function and just reuse it by way of calling it.